ldk/javascript/examples/self-test-loop/src/tests/network/index.ts

```diff
@@ -121,31 +121,19 @@ export const testWebsocketConnection = (): Promise<boolean> =>
         const socket = await network.webSocketConnect(socketConfiguration);
         console.info('Websocket successfully connected');
 
-        await socket.setCloseHandler((error, code, text) => {
-          if (error) {
-            console.error('setCloseHandler error', error);
-            reject(error);
-          }
-
-          console.info(`Received on close code: ${code}. ${text}`);
-
-          // This is from us closing the socket connection in the message handler below
-          if (testPassed) {
-            resolve(true);
-          }
-        });
-
+        socket.writeMessage('You are using a test api key');
```

Comment:
I think we should do `await socket.writeMessage('You are using a test api key');` since `writeMessage` returns a `Promise`